Framework-User-Guide.md

```diff
@@ -0,0 +1,90 @@
+![logo](https://github.com/culturegraph/metafacture-core/wiki/img/metafacture_small.png)
+
+# Framework User Guide
+
+> [!NOTE]
+>Relevant for JAVA developers. For using metafacture without JAVA Code see the [FLUX user guide](/Flux-User-Guide.md).
+
+This page explains how to create a Metafacture objects and how to assemble them to form a processing pipeline. We use as an example a simple pipeline containing a Metamorph instance.
+
+
+## Building a Flow
+
+A Flow consists of a data source, an arbitrary number of pipe elements and finally a data sink.
+The individual elements are connected by calling the `setReceiver()` method. The following code snipped shows an example.
+
+```java
+// create necessary objects
+final PicaReader reader = new PicaReader();
+final Metafix metafix = new Metafix("defnition.fix");
+final ListMapWriter writer = new ListMapWriter();
+
+//connect them
+reader.setReceiver(fix).setReceiver(writer);
+
+//start processing
+reader.read(input);
```

Comment:
Is it "read()" or "process()" ? I am not sure.

Reply:
This is from the old MF Documentation.

Reply:
perhapse @fsteeg  can help

Reply:
It should be `process`, see https://github.com/metafacture/metafacture-core/blob/9cf783f91487b1f8637a475ca87db8e4e58122e9/metafacture-framework/src/main/java/org/metafacture/framework/ObjectReceiver.java#L40